pizza-time/src/components/login/login.js

```diff
@@ -1,11 +1,96 @@
-import React from 'react';
+import React, { useReducer } from "react";
+import firebaseApp from "../../firebase/firebaseApp";
 
-const Login = () => {
-    return(
+//need this import for "firebase.auth.Auth.Persistence.LOCAL" constant
```

Comment:
Looks good, surprised you got this up so quickly

Reply:
Thanks, I appreciate it man. I wanted us to be ahead of the game.